File: src/main/java/com/aquasecurity/plugins/trivy/model/Result.java
package com.aquasecurity.plugins.trivy.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class Result {
    @JsonProperty("Target")
    public String target;
    @JsonProperty("Class")
    public String ClassName;
    @JsonProperty("Type")
    public String type;
    @JsonProperty("Vulnerabilities")
    public List<Vulnerability> vulnerabilities;
    @JsonProperty("MisconfSummary")
    public MisconfSummary misconfSummary;
    @JsonProperty("Misconfigurations")
    public List<Misconfiguration> misconfigurations;
}

File: src/main/kotlin/com/aquasecurity/plugins/trivy/model/Result.kt
package com.aquasecurity.plugins.trivy.model

import com.fasterxml.jackson.annotation.JsonProperty

class Result {
    @JsonProperty("Target")
    var target: String? = null

    @JsonProperty("Class")
    var ClassName: String? = null

    @JsonProperty("Type")
    var type: String? = null

    @JsonProperty("Vulnerabilities")
    var vulnerabilities: List<Vulnerability>? = null

    @JsonProperty("MisconfSummary")
    var misconfSummary: MisconfSummary? = null

    @JsonProperty("Misconfigurations")
    var misconfigurations: List<Misconfiguration>? = null

    @JsonProperty("Secrets")
    var secrets: List<Secret>? = null
}